docs/tutorial/visualization/open3d_visualizer_advanced.rst

```diff
@@ -0,0 +1,511 @@
+.. _open3d_visualizer_advanced:
+
+Advanced Open3D Visualizer
+==========================
+
+Introduction
+---------------
+
+.. epigraph:: Open3D provides a convenient function for visualizing geometric objects: ``draw``. The ``draw`` function allows you to visualize multiple geometry objects *(PointClouds, LineSets, TriangleMesh)* and images together along with optional, high-quality, physically based (PBR) materials. As will be demonstrated in the subsequent sections, ``draw`` can be used for both - simple, quick visualization or complex use-cases.
```

Comment:
Is this the same as in the basic tutorial? Instead, ask them to check out the basic tutorial first.

Reply:
Changed last sentences in the Introduction. Added a link to the Basic o3d Visualizer Tutorial.